src/main/java/Bellroy/Todo.java

```diff
@@ -0,0 +1,18 @@
+package Bellroy;
+
+/**
+ * Encapsulates a Todo Task
+ */
+public class Todo extends Task {
+
+    public Todo(String description) {
+        super("T", description);
+    }
+
+    @Override
+    public String toString() {
+        String message = "[" + this.type + "]" + "[" + this.getStatusIcon() + "] " + this.description;
```

Comment:
I used StringBuilder() for String that has multiple concatenation operation as it would be more optimised. Maybe you can also try using it? Do let me know your thoughts! :)